package.json

```diff
@@ -4,7 +4,10 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "start:dev": "nodemon ./bin/www"
+    "start:dev": "nodemon ./bin/www",
+    "migrate:reset": "sequelize db:migrate:undo:all && sequelize db:migrate",
+    "test": "NODE_ENV=test jest --setTimeout=10000 --detectOpenHandles",
```

Comment:
Any particular reason for the `setTimeout` argument?

Reply:
Haha, no that should be `--testTimeout`, there is no `--setTimeout` option. The idea is to make it big enough that Jest doesn't timeout when it resets the database between tests, but clearly not setting it doesn't seem to be causing any problems.

Reply:
Ah, it likely has a default timeout value that we're falling within then?

Reply:
Yes, apparently the default is 5000. I'll just remove that flag